setup.py

```diff
@@ -294,11 +294,63 @@ def adjust_installation_command():
                 else:
                     readme_lines[ln] = "  pip install Pootle\n"
 
+    def replace_urls():
+        from pootle.core.utils import version
+        branch_escape = version.get_git_branch().replace('/', '%2F')
+        for ln, line in enumerate(readme_lines):
+            for pattern, replace, rewrite_type in (
+                # Release Notes
+                ('releases/[0-9]\.[0-9]\.[0-9]\.html', 'releases/%s.html' %
+                 version.get_main_version(), 'all'),
+                # Adjust docs away from /latest/
+                ('/pootle/en/latest/', '/pootle/en/%s/' %
+                 version.get_rtd_version(), 'branch'),
+                # Coverage - Codecov for branches
+                ('codecov.io/gh/translate/pootle/branch/master',
+                 'codecov.io/gh/translate/pootle/branch/%s' % branch_escape,
+                 'branch'),
+                ('shields.io/codecov/c/github/translate/pootle/master',
+                 'shields.io/codecov/c/github/translate/pootle/%s' %
+                 branch_escape, 'branch'),
+                # Coverage - Coveralls for tags
+                ('codecov.io/gh/translate/pootle/branch/master',
+                 'coveralls.io/github/translate/pootle?branch=%s' %
+                 version.get_version(), 'tag'),
+                ('shields.io/codecov/c/github/translate/pootle/master',
+                 'shields.io/coveralls/translate/pootle/%s' %
+                 version.get_version(), 'tag'),
+                # Travis - change only the badge, can't link to branch
+                ('travis/translate/pootle/master', 'travis/translate/pootle/%s'
+                 % version.get_git_branch(), 'branch'),
+                ('travis/translate/pootle/master', 'travis/translate/pootle/%s'
+                 % version.get_version(), 'tag'),
+                # Landscape
+                ('landscape.io/github/translate/pootle/master',
+                 'landscape.io/github/translate/pootle/%s' %
+                 version.get_git_branch(), 'branch'),
+                # Requires.io
+                ('requires/github/translate/pootle/master',
+                 'requires/github/translate/pootle/%s' %
+                 version.get_git_branch(), 'branch'),
+                ('requirements/\?branch=master', 'requirements/?branch=%s' %
+                 branch_escape, 'branch'),
+                ('https://img.shields.io/requires/.*',
+                 'https://requires.io/github/translate/pootle/requirements.svg?tag=%s'
+                 % version.get_version(), 'tag'),
+                ('requirements/\?branch=master', 'requirements/?tag=%s' %
+                 version.get_version(), 'tag'),
+            ):
```

Comment:
Nit: put this tuple of tuples out of the `for` loops to increase code readability.